components/SettingsControls/src/SettingsExpander/SettingsExpander.xaml

```diff
@@ -99,7 +99,7 @@
                                     <ContentPresenter Content="{TemplateBinding ItemsHeader}" />
                                     <muxc:ItemsRepeater x:Name="PART_ItemsRepeater"
                                                         Grid.Row="1"
-                                                        ItemTemplate="{Binding ItemTemplate, Mode=OneWay, RelativeSource={RelativeSource TemplatedParent}}"
+                                                        ItemTemplate="{TemplateBinding ItemTemplate}"
```

Comment:
@YourOrdinaryCat was there a reason for this change? I know we used the longform here in case the template is changed dynamically, as the shortcut doesn't do well with changes always.